website/meta/universe.json

```diff
@@ -1257,6 +1257,45 @@
             "category": ["pipeline", "models", "training"],
             "tags": ["pipeline", "models", "transformers"]
         },
+        {
+            "id": "asent",
+            "title": "Asent",
+            "slogan": "Fast, flexible and transparent sentiment analysis",
+            "description": "Asent is a rule-based sentiment analysis library for Python made using SpaCy. It is inspired by Vader, but uses a more modular ruleset, that allows the user to change e.g. the method for finding negations. Furthermore it includes visualiser to visualize model predictions, making they model easily interpretable.",
```

Comment:
Very minor point, but VADER is an abbreviation and should be capitalized.

Also the last sentence should be:

> Furthermore it includes **a** visualizer, to make **the** model easily interpretable.

Reply:
also "spaCy"

Reply:
Thanks for these corrections, they have now been added.